conda_build/environ.py

```diff
@@ -469,8 +469,9 @@ def linux_vars(compiler_vars, prefix, config):
     if config.arch == 32:
         compiler_vars['CFLAGS'] += ' -m32'
         compiler_vars['CXXFLAGS'] += ' -m32'
-    return {}
```

Comment:
This is not your bug, but surely this is not right?  It's always returning an empty dict?  I guess we get away with it because we're modifying compiler_vars in place here, but this code smells.

I'll add a separate issue for this code smell.